File: Python/tests/test_prl_fictitious_cnc.py
import pytest

from hbayesdm.models import prl_fictitious_cnc


def test_prl_fictitious_cnc():
    _ = prl_fictitious_cnc(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_prl_fictitious_cnc.R
context("Test prl_fictitious_cnc")
library(hBayesDM)

test_that("Test prl_fictitious_cnc", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(prl_fictitious_cnc(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
